client/my-sites/domains/domain-management/list/index.jsx

```diff
@@ -437,23 +355,25 @@ export class List extends React.Component {
 				this.setState( {
 					settingPrimaryDomain: false,
 					changePrimaryDomainModeEnabled: false,
-					notice: {
-						type: PRIMARY_DOMAIN_CHANGE_SUCCESS,
-						domainName: domain.name,
-						previousDomainName: currentPrimaryName,
-					},
 				} );
+
+				this.props.successNotice(
+					translate(
+						'Primary Domain changed: All domains will redirect to {{em}}%(domainName)s{{/em}}.',
```

Comment:
I know this is from the Asana card, but the capitalization here is bothering me a bit. Maybe let's keep it simple a say `Primary domain changed: all domains will redirect to {{em}}%(domainName)s{{/em}}.`?

Reply:
good suggestion. I like your suggestion Igor!